dcs/mission.py

```diff
@@ -1808,6 +1808,21 @@ def find_group(self, group_name, search="exact") -> Optional[unitgroup.Group]:
                 return g
         return None
 
+    def find_group_by_id(self, group_id: int) -> Optional[unitgroup.Group]:
+        """Searches a group with the given groupId
+
+        Args:
+            group_id: group identifier assigned by the mission file
+
+        Returns:
+            Group: the group found, otherwise None
+        """
+        for k in self.coalition:
+            g = self.coalition[k].find_group_by_id(group_id)
+            if g:
```

Comment:
`is not None`

Reply:
@DanAlbert - g is an object, it cannot be anything else but Group or None. Do you want to be that explicit?

Reply:
But that doesn't mean it can't be false. `__bool__` is a thing.

Reply:
Added is not None.